keyboards/arabica37/keymaps/default/keymap.c

```diff
@@ -0,0 +1,173 @@
+    KC_TRNS,                        KC_TRNS,KC_TRNS,           KC_TRNS,KC_TRNS,                        KC_TRNS,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------|
+                                    KC_TRNS,                           KC_TRNS
+ //`-----------------------------------------------'          `-----------------------------------------------'
+ ),
+ [_RAISE] = LAYOUT( \
+ //,-----------------------------------------------.          ,-----------------------------------------------.
+    KC_TRNS,KC_EXLM,  KC_AT,KC_HASH, KC_DLR,KC_PERC,           KC_CIRC,KC_PSLS,KC_PAST,KC_PMNS,KC_PPLS,KC_TRNS,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------|
+    KC_TRNS,KC_RBRC,KC_LABK,KC_ASTR,KC_SCLN,KC_LBRC,              KC_1,   KC_2,   KC_3,   KC_4,   KC_5, KC_EQL,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------|
+    KC_TRNS,KC_BSLS,KC_RABK,KC_LPRN,KC_JYEN,KC_UNDS,              KC_6,   KC_7,   KC_8,   KC_9,   KC_0,KC_PENT,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+--- ---+-------+-------|
+    KC_TRNS,                        KC_TRNS,KC_TRNS,           KC_TRNS,KC_TRNS,                        KC_TRNS,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+--- ---+-------+-------|
+                                    KC_TRNS,                           KC_TRNS
+ //`-----------------------------------------------'          `-----------------------------------------------'
+ ),
+ [_ADJUST] = LAYOUT( \
+ //,-----------------------------------------------.          ,-----------------------------------------------.
+    KC_TRNS, RGBRST,RGB_TOG,  KC_NO,  KC_NO,  KC_NO,             KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------|
+    KC_TRNS,RGB_HUI,RGB_SAI,RGB_VAI,AG_SWAP,  KC_NO,             KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------|
+    KC_TRNS,RGB_HUD,RGB_SAD,RGB_VAD,AG_NORM,  KC_NO,             KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------|
+    KC_TRNS,                        KC_TRNS,KC_TRNS,           KC_TRNS,KC_TRNS,                        KC_TRNS,\
+ //|-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------|
+                                    KC_TRNS,                           KC_TRNS
+ //`-----------------------------------------------'          `-----------------------------------------------'
+ )
+};
+
+int RGB_current_mode;
+
+void matrix_init_user(void) {
```

Comment:
This really should be: 
```suggestion
void keyboard_post_init_user(void) {
```

That way, it has the properly loaded info.